lib/auth.ts

```diff
@@ -462,26 +461,72 @@ const getAuth = function (
 	 * });
 	 *
 	 * @example
+	 * // If email verification is enabled
```

Comment:
How about having a single register call and inside the `then` have an `if` statement that does a `console.log('registrered)` or `console.log('Email verification is required')`?